modules/apps/transfer/keeper/relay_test.go

```diff
@@ -250,7 +383,22 @@ func (suite *KeeperTestSuite) TestOnRecvPacket() {
 			data := types.NewFungibleTokenPacketData(trace.GetFullDenomPath(), amount.String(), suite.chainA.SenderAccount.GetAddress().String(), receiver, memo)
 			packet := channeltypes.NewPacket(data.GetBytes(), seq, path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, path.EndpointB.ChannelConfig.PortID, path.EndpointB.ChannelID, clienttypes.NewHeight(1, 100), 0)
 
-			err = suite.chainB.GetSimApp().TransferKeeper.OnRecvPacket(suite.chainB.GetContext(), packet, data)
+			suite.Require().NotPanics(func() {
+				err = suite.chainB.GetSimApp().TransferKeeper.OnRecvPacket(suite.chainB.GetContext(), packet, data)
+			})
+
+			// check total amount in escrow of received token denom on receiving chain
+			var denom string
+			var totalEscrow math.Int
+			if tc.recvIsSource {
+				denom = sdk.DefaultBondDenom
+			} else {
+				denom = types.ParseDenomTrace(
+					types.GetPrefixedDenom(path.EndpointB.ChannelConfig.PortID, path.EndpointB.ChannelID, sdk.DefaultBondDenom),
+				).IBCDenom()
```

Comment:
Because the test exercises `OnRecvPacket` and in that callback it will update the total escrow amount for the received denom in case the receiver chain is the source. Since the received tokens will have an IBC denom and the receiving chain is not the source, then we expect that no update will happen.